node/src/components/gossiper/metrics.rs

```diff
@@ -44,42 +43,42 @@ impl GossiperMetrics {
                 name
             ),
         )?;
-        let tbl_items_paused = IntGauge::new(
-            format!("{}_tbl_items_paused", name),
+        let table_items_paused = IntGauge::new(
+            format!("{}_table_items_paused", name),
             format!(
                 "number of items in the gossip table of {} in state paused",
                 name
             ),
         )?;
-        let tbl_items_current = IntGauge::new(
-            format!("{}_tbl_items_current", name),
+        let table_items_current = IntGauge::new(
+            format!("{}_table_items_current", name),
             format!(
                 "number of items in the gossip table of {} in state current",
                 name
             ),
         )?;
-        let tbl_items_finished = IntGauge::new(
-            format!("{}_tbl_items_finished", name),
+        let table_items_finished = IntGauge::new(
+            format!("{}_table_items_finished", name),
```

Comment:
Thanks for changing `tbl` to `table`. I didn't want to pick on that.

Reply:
I couldn't resist :smile:  Seemed worth it for the sake of 2 chars.